Configurations/Plists/GoogleService/Production/GoogleService-Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>814505320278-tesgb721mamm7vmrppnpp8mm4jqdjql0.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
```

Comment:
FYI, I can't make the `Google Services Plist` files visible in side the project because I have an archive issue from [here](https://github.com/nimblehq/nimble-crypto-ios/actions/runs/3581691977/jobs/6025043926):

![Screenshot 2022-11-30 at 15 57 14](https://user-images.githubusercontent.com/70877098/204752239-0dc493ac-9678-4aec-ab18-0bf343d1d8ee.png)

-> As a result, only the `Configurations` folder is visible. 🙏